src/apps/TrafimageWKP/README.md

```diff
@@ -1,14 +1,8 @@
 #
 
-Configure your own map.
-
-'TrajservLayer' Layer filter options (case insensitive):
+The Trafimage Web Map Portal is used to illustrate topics relating to public transport.
```

Comment:
Trafimage maps are used to illustrate a whole range of topics relating to public transport ([more information](https://www.sbb.ch/en/bahnhof-services/bahnhoefe/karten-bahnhofplaene/trafimage-karten.html)). 
This section shows you how to configure your own map for integrating it in your existing web page.